README.md

```diff
@@ -3,30 +3,62 @@
 
 This repository includes the recipe for deploying a Tella Web instance (frontend and backend).
 
+## System requirements (recommended) 
```

Comment:
Don't we want to say that requirements (specially disk space) depend on the data collected? Maybe we could say that this is the minimum setup and that we recomend to monitor the server (and maybe even link to meerkat if we want) and adjust it based on usage.